src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -45,12 +47,18 @@ function ExpensifyCardPage({
     const virtualCard = _.find(domainCards, (card) => card.isVirtual) || {};
     const physicalCard = _.find(domainCards, (card) => !card.isVirtual) || {};
 
-    if (_.isEmpty(virtualCard) && _.isEmpty(physicalCard)) {
+    const [shouldShowCardDetails, setShouldShowCardDetails] = useState(false);
+
+    if (_.isEmpty(virtualCard) || _.isEmpty(physicalCard)) {
```

Comment:
We need to have here` if (_.isEmpty(virtualCard) && _.isEmpty(physicalCard)) {`

Reply:
Done